source/includes/steps-export-pipeline-to-language.yaml

```diff
@@ -42,20 +42,33 @@ content: |
      selected.
 ---
 title: Include import statements, if desired.
+optional: true
 level: 4
 ref: include-import-statements
 content: |
-  *(Optional)*: Check the :guilabel:`Include Import Statements` option
+  Check the :guilabel:`Include Import Statements` option
   to include the required import statements for the language selected.
 ---
-title: Click :guilabel:`Copy`.
+title: Include driver syntax, if desired.
+optional: true
+level: 4
+ref: include-driver-syntax
+content: |
+  Check the :guilabel:`Include Driver Syntax` option to include 
+  application code for the language selected. If you include driver 
+  syntax, the copyable code reflects :ref:`project 
```

Comment:
[nit] perhaps:
- "reflects the configured X, Y, and Z options",  or 
- "reflects your configured X, Y, and Z options"

Would it be useful to indicate where I configured these options? i.e.

- "reflects the X, Y, and Z options as configured in <Fantastic_Admin_Panel_5>"